### PR TITLE
Update tcp.go

### PR DESCRIPTION
**Pull request by ruppde**
workaround to not wait for Interval seconds to get the 1st ping

### CODE REVIEW

File: ping/tcp.go [hunk 1]
```diff
@@ -39,11 +39,15 @@ func (tcping TCPing) Result() *Result {
...
 	go func() {
-		t := time.NewTicker(tcping.target.Interval)
+		// cheap workaround to not wait for Interval seconds to get the 1st ping: set short wait time and ...
+		t := time.NewTicker(1)
 		defer t.Stop()
 		for {
 			select {
 			case <-t.C:
+				// ... change to real Interval after the 1st run
+				t = time.NewTicker(tcping.target.Interval)
```
Comment (cloverstd) [inline ping/tcp.go:49]:
```suggestion
				t.Stop()
				t = time.NewTicker(tcping.target.Interval)
```

Comment (cloverstd) [reply ping/tcp.go:49]: The ticker should be stopped.

Comment (2d4d) [reply ping/tcp.go:49]: doesn't work for me, doesn't pause anymore:

$ time ~/go/bin/tcping -I 2 ix.de
Ping tcp://ix.de:80(193.99.144.80:80) - Connected - time=11.652924ms
Ping tcp://ix.de:80(193.99.144.80:80) - Connected - time=10.546808ms
Ping tcp://ix.de:80(193.99.144.80:80) - Connected - time=9.977961ms
Ping tcp://ix.de:80(193.99.144.80:80) - Connected - time=10.407401ms

Ping statistics tcp://ix.de:80
	4 probes sent.
	4 successful, 0 failed.
Approximate trip times:
	Minimum = 9.977961ms, Maximum = 11.652924ms, Average = 10.646273ms

real	0m0,053s
user 0m0,007s
sys	0m0,012s

Comment (cloverstd) [reply ping/tcp.go:49]: the `time.NewTicker` will add timer to the P's heap.

https://github.com/golang/go/blob/master/src/runtime/time.go#L277-L295

It should be stopped to release resource.

Comment (2d4d) [reply ping/tcp.go]: Reset() should fix that, see updated PR

**Pull request merged**